packages/components/src/chart-uplot/index.tsx

```diff
@@ -86,6 +88,7 @@ export default function UplotChart( {
 						ticks: {
 							show: false,
 						},
+						filter: yAxisFilter,
```

Comment:
I tried the [axis option incrs](https://github.com/leeoniya/uPlot/blob/b07903043863d2eb784f1f6fab19fa77d12f5637/dist/uPlot.d.ts#L963-L964), and It seems we can get the same effect by setting `incrs: [ 0, 1 ],`.